lib/vsc/install/commontest.py

```diff
@@ -137,10 +137,23 @@ class CommonTest(TestCase):
     ]
 
     # Prospector commandline options (positional path is added automatically)
+    prospector_profile = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+        'etc', 'pylint_py3_checker.yaml')
+    # The prospector_profile (in theory) should be in `lib/etc` in the distribution file and
+    # in `etc` after installation. (Chosen in a way to be the same place realtively to shared_setup.py
+    if not os.path.exists(prospector_profile):
+        log.error("File prospector_profile: '%s' not found" % prospector_profile)
```

Comment:
no need to report the same thing you raise.
(and if you log, no need for templating)